zero/src/rpc/jerigon.rs

```diff
@@ -33,16 +49,63 @@ where
             "debug_traceBlockByNumber".into(),
             (target_block_id, json!({"tracer": "zeroTracer"})),
         )
-        .await?;
+        .await?
+        .into_iter()
+        .map(|ztr| ztr.result)
+        .collect::<Vec<_>>();
 
     // Grab block witness info (packed as combined trie pre-images)
-
     let block_witness = cached_provider
         .get_provider()
         .await?
         .raw_request::<_, String>("eth_getWitness".into(), vec![target_block_id])
         .await?;
 
+    let block = cached_provider
+        .get_block(target_block_id, BlockTransactionsKind::Full)
+        .await?;
+
+    let tx_traces: Vec<&BTreeMap<__compat_primitive_types::H160, TxnTrace>> = tx_results
+        .iter()
+        .map(|TxnInfo { traces, meta: _ }| traces)
+        .collect::<Vec<_>>();
+
+    let jdts: Vec<Option<JumpDestTableWitness>> = process_transactions(
```

Comment:
>why don't we include it in the zero tracer witness output

I agree this would indeed be a neater.

Reply:
I also agree, this was discussed on slack and Jerry suggested to implement it directly instead of modifying the Zero tracer.

Reply:
Should we than wait with this change until it is implemented in Jerigon?

Reply:
I don't think it will be done anytime soon, because Jerry doesn't have any bandwidth for this. Given that we still support the pre-existing simulation by the prover to retrieve the jump destinations, we could just have it optional through some CLI arg, as it would still be beneficial for the native tracer case. Then if we end up implementing it for Jerigon in the zero tracer itself we'll rely on that one.

Reply:
Having it with some optional CLI flag and with native tracer is totally fine. We already have 3 rpc calls for every transaction (2 `debug_trace_transaction` )with native tracer, adding the 4th one is not too big difference.

Reply:
I will try adding a CLI flag with default depending on tracer type.

Reply:
Done, see latest commits.